pkg/reconciler/domainmapping/table_test.go

```diff
@@ -1533,6 +1545,36 @@ func patchRemoveFinalizerAction(namespace, name string) clientgotesting.PatchAct
 	}
 }
 
+func v1ServiceInformer(ctx context.Context, objects []runtime.Object) corev1informers.ServiceInformer {
+	kubeInformerFactory := kubeinformers.NewSharedInformerFactory(
+		fakekubeclient.NewSimpleClientset(objects...), 0)
+	svcInformerIface := kubeInformerFactory.Core().V1().Services()
+	svcInformerIface.Informer() // register informer in factory
+	go kubeInformerFactory.Start(ctx.Done())
+	kubeInformerFactory.WaitForCacheSync(ctx.Done())
+	return svcInformerIface
+}
```

Comment:
We have a way to pass fake listers around - we don't need to create and start fake informers. Why doesn't the `resolver.NewURIResolverFromTracker` accept a service lister?